README.md

```diff
@@ -127,12 +134,12 @@ Your `Staker UI` tab should be almost done and working at this point.
 #### 🥅 Goals
 
 - [ ] Can you see `timeLeft` counting down in the `Staker UI` tab when you trigger a transaction with the faucet button?
-- [ ] If you `stake()` enough ETH before the `deadline`, does it call `complete()`?
-- [ ] If you don't `stake()` enough can you `withdraw()` your funds?
+- [ ] If enough Eth is staked by the deadline, does your `execute()` function correctly call `complete()` and stake the Eth?
```

Comment:
Sorry about this one, very nitpicky, don't hate me! Think we are using ETH in many places of the Readme, I don't have any special preference, but I think we should use the same term in all the document, either ETH or Eth

Reply:
Good catch.  It has been updated to ETH to match the others.